### PR TITLE
Refactor dtc test to actually require DTC support

### PR DESCRIPTION
**Pull request by andreasohlund**
I have verified that this test runs green on SQLT and NH and that it escalates to DTC

### CODE REVIEW

File: src/NServiceBus.Learning.AcceptanceTests/NServiceBus.Learning.AcceptanceTests.csproj [hunk 1]
```diff
@@ -19,6 +19,7 @@
   <ItemGroup>
     <Compile Include="../NServiceBus.AcceptanceTests/**/*.cs" Exclude="../NServiceBus.AcceptanceTests/obj/**/*.*;../NServiceBus.AcceptanceTests/Core/**/*.cs" />
     <Compile Remove="../NServiceBus.AcceptanceTests/Core/TestSuiteConstraints.cs" />
+    <Compile Remove="..\NServiceBus.AcceptanceTests\Tx\Issue_2481.cs" />
```
Comment (bording) [inline src/NServiceBus.Learning.AcceptanceTests/NServiceBus.Learning.AcceptanceTests.csproj:22]: Why is this change here?

Comment (andreasohlund) [reply src/NServiceBus.Learning.AcceptanceTests/NServiceBus.Learning.AcceptanceTests.csproj]: Fixed

**Comment by andreasohlund** [comment]
I've now been through the tests that are flagged as requiring DTC and removed/refactored the ones that didn't need it

**Review by kentdr — APPROVED** [review]
